posts/Davide/2015-06-02-hibernate-ogm-42-final-is-out.html.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: "Hibernate OGM 4.2 Final is out"
+author: "Davide D'Alto"
+blogger_name: "Davide"
+creation_date: "02-06-2015"
+original_tags: [Hibernate,Hibernate OGM,News]
+tags: [Hibernate OGM]
+
+relative_url: /2015/06/02/hibernate-ogm-42-final-is-out
+slug: hibernate-ogm-42-final-is-out
+lace: http://in.relation.to/42822.lace
+
+layout: blog-post
+
+disqus_thread_id: http://in.relation.to/2015/06/02/hibernate-ogm-42-final-is-out
+---
+<div id="documentDisplay" class="documentDisplay">
+
+
+<p class="wikiPara">
```

Comment:
Is there a reason for keeping the legacy CSS classes or should we enhance the importer remove them? We don't style them anyways, so they only increase the size of the delivered HTML. Thoughts? Can be done separately of course.